reviewpad.yml

```diff
@@ -0,0 +1,71 @@
+api-version: reviewpad.com/v3.x
+
+labels:
+  size/xs:
+    color: '5D9801'
+    description: 'Denotes a change 0-9 lines'
+  size/S:
+    color: '#5D9801'
+    description: 'Denotes a change 10-29 lines'
+  size/M:
+    color: '7F7203'
+    description: 'Denotes a change 30-99 lines'
+  size/L:
+    color: 'A14C05'
+    description: 'Denotes a change 100-499 lines'
+  size/XL:
+    color: 'C32607'
+    description: 'Denotes a change 500-999 lines'
+  size/XXL:
+    color: 'E50009'
+    description: 'Denotes a change 1000+ lines'
+
+workflows:
+  - name: label-pull-request-with-size
+    if:
+      - rule: $size() <= 9
+        extra-actions:
+          - $addLabel("size/XS")
+      - rule: $size() => 10 && $size() <= 29
```

Comment:
This rule has a parsing error. The sign `=>` should be `>=`. Like you have it [here](https://github.com/schmelto/Portfolio/pull/224/files#diff-d7a202cc076357dca8f9a8931b252f8ba4803ea0d12b140e62bea228410ad634R41). The same applies to the conditions bellow 😄

Reply:
Thanks great spot